sqlalchemy_jsonapi/declarative/serializer.py

```diff
@@ -0,0 +1,82 @@
+"""A serializer for serializing SQLAlchemy models to JSON API spec."""
+
+
+class JSONAPISerializer(object):
+    """A JSON API serializer that serializes SQLAlchemy models."""
+
```

Comment:
Please note that there is not an `__init__`. I was not sure if we needed one at this point. Particularly because errors that we would raise when be called when we instantiated the class. However if we are to use this in say a flask view such as:
```
# Some GET view in web framework such as Flask
def get_resource(db_id):
    # Fetch some resource from database.
    some_sqlalchemy_model_resource = db.query(Person).get(db_id)
    person_serializer = PersonSerializer()
    data = person_serializer.serialize(some_sqlalchemy_model_resource)
    return flask.jsonify(data), status_code
```
If we were to raise exceptions in `__init__` we would then have to have someone except those errors when instantiating the PersonSerializer. 

Because I was not sure I left these checks to be handled when `serialize` is called instead.
I will note these checks in code below.